solidity/contracts/converter/types/standard-pool/StandardPoolConverter.sol

```diff
@@ -997,7 +998,7 @@ contract StandardPoolConverter is ConverterVersion, IConverter, ContractRegistry
         // transfer the fees and sync the balances to ensure no mismatch
         (oldReserveBalances[0], oldReserveBalances[1]) = processNetworkFees(0);
 
-        uint256[] memory reserveAmounts = removeLiquidityReserveAmounts(_amount, totalSupply, oldReserveBalances);
+        uint256[2] memory reserveAmounts = removeLiquidityReserveAmounts(_amount, totalSupply, oldReserveBalances);
 
         for (uint256 i = 0; i < 2; i++) {
```

Comment:
no need for all these for loops if there are only 2.

Reply:
Yes, but there are 7 lines of code inside the loop. Do you actually want to replicate them instead of using a `for` loop?